modules/misc/xdg-mime-apps.nix

```diff
@@ -75,14 +85,9 @@ in
   };
 
   config = mkIf cfg.enable {
-    xdg.configFile."mimeapps.list".text =
```

Comment:
I think it would be best to keep this as-is and instead simply have
```
# Deprecated but still used by some applications.
home.file.".local/share/applications/mimeapps.list".source =
  xdg.configFile."mimeapps.list".source;
```

Reply:
Nice idea, fixed it!